presto-benchmark/src/main/java/com/facebook/presto/benchmark/SqlConsecutiveJoinBenchmark.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark;
+
+import com.facebook.presto.testing.LocalQueryRunner;
+
+import static com.facebook.presto.benchmark.BenchmarkQueryRunner.createLocalQueryRunner;
+
+public class SqlConsecutiveJoinBenchmark
+        extends AbstractSqlBenchmark
+{
+    public SqlConsecutiveJoinBenchmark(LocalQueryRunner localQueryRunner)
+    {
+        super(localQueryRunner,
+                "sql_consecutive_join",
+                4,
+                5,
+                "SELECT * FROM\n" +
+                        "(\n" +
```

Comment:
This nice plan nesting looks like you mean a specific query plan (joining tree) shape. Is it guaranteed? 
See `com.facebook.presto.SystemSessionProperties#REORDER_JOINS`